<!-- FingerVision demo of two video streams (no sensors) over Ethernet network.
Before executing this launch, stream videos by:
  $ rosrun fingervision stream_file1.sh
-->
<launch>
  <node type="fv_core_suzuki_node" pkg="fingervision" name="rh_l" ns="fingervision" output="screen">
    <param name="pkg_dir" value="$(find fingervision)/" />
    <param name="cam_config" value="config/rh_l.yaml" />
    <param name="stereo_config" value="config/rh_l.yaml" />
    <param name="blobtrack_config" value="config/rh_l.yaml" />
    <param name="objdettrack_config" value="config/rh_l.yaml" />
    <param name="blob_calib_prefix" value="data_gen/blob_" />
    <param name="frame_skip" value="0" />
    <param name="target_fps" value="30" />
  </node>
</launch>
